x-pack/legacy/plugins/reporting/export_types/png/types.d.ts

```diff
@@ -8,7 +8,7 @@ import { Request } from 'hapi';
 import { LayoutInstance } from '../common/layouts/layout';
 import { ConditionalHeaders, KbnServer, JobDocPayload } from '../../types';
 
-// NOTE: this does not extend the main Params
```

Comment:
The comment was removed because it is moot - none of the type extend from the main params, so there probably shouldn't even be a main params. Separate issue.